packages/cli/src/util/update-notifier/update-worker.js

```diff
@@ -20,7 +20,14 @@ process.on('unhandledRejection', err => {
   process.exit(1);
 });
 
-process.on('message', async msg => {
+// this timer will prevent this worker process from running longer than 10s
+const timer = setTimeout(() => {
```

Comment:
**suggestion (non-blocking):** rename `timer` -> `workerCompleteTimer`

Can we use a more specific name for the timeout handle? It will make it clearer what the `clearTimeout` line is doing, for example:

```
clearTimeout(workerCompleteTimer);
```